src/components/form/form.tsx

```diff
@@ -41,11 +42,21 @@ export class Form {
     @Event()
     public change: EventEmitter<object>;
 
+    /**
+     * Emitted when the validity of the form changes
```

Comment:
> Emitted when the validity of the form changes

That's not quite true as far as I can see. If I read the code correctly, it will be emitted any time the validity changes, but also any time the form is updated and it fails validation, regardless of whether it was already invalid before the update.

Not that we necessarily have to specify to that level of detail. I just wanted to check that I've read the code correctly, and that the code is as intended? I assume we need to send the event whenever the form is invalid, regardless of prior state, because the error messages might have changed?

Reply:
Yeah that was what I intended, I think I forgot to push the updated docstring, becuase I'm quite sure I wrote an update to it at some point :smile: 
But you agree that it should work that way? It seems correct, right?

Reply:
Yes, that seems correct to me 👍